### PR TITLE
set the environment variables

### PR DESCRIPTION
**Pull request by kelvin13**
right now this script doesn’t work at all; also made the output a bit more readable

### CODE REVIEW

File: src/sample.py [hunk 1]
```diff
@@ -2,21 +2,29 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function
+
+from os import environ
+environ['LD_LIBRARY_PATH'] = '/usr/local/lib/girepository-1.0'
+environ['GI_TYPELIB_PATH'] = '/usr/local/lib/girepository-1.0'
```
Comment (khaledhosny) [inline src/sample.py]: This is only needed if HarfBuzz is installed there, which is not always the case.

Comment (kelvin13) [inline src/sample.py]: this is the default install directory, and in fact the *only* install directory that harfbuzz works in, see #274

**Comment by behdad** [comment]
First, no mixing of two separate issues into one PR.

Second, env-vars don't belong here.  It's not "the _only_ install directory that harfbuzz works in".  I always use ~/.local for example.

**Comment by kelvin13** [comment]
trying to install harfbuzz in a custom directory never worked for me, you get sporadic failures & `Failed to load shared library 'libharfbuzz-gobject.so.0' referenced by the typelib: libharfbuzz-gobject.so.0: cannot open shared object file: No such file or directory`

plus it should at least be worth it to put "`<your girepository folder location here>`" or something so people realize that’s why it’s not working out when they download this file

**Comment by behdad** [comment]
> trying to install harfbuzz in a custom directory never worked for me, you get sporadic failures & Failed to load shared library 'libharfbuzz-gobject.so.0' referenced by the typelib: libharfbuzz-gobject.so.0: cannot open shared object file: No such file or directory

That's because you failed to set `LD_LIBRARY_PATH`, but that's basic Linux / Unix issue, not HarfBuzz-related.

**Comment by kelvin13** [comment]
I did set LD_LIBRARY_PATH

**Comment by khaledhosny** [comment]
Did you try to set the environment variable before calling python?

**Comment by kelvin13** [comment]
what do you mean before? I thought you set the environment variable in the code

**Pull request closed without merging**